content/en/tracing/connect_logs_and_traces/ruby.md

```diff
@@ -17,17 +17,20 @@ further_reading:
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Automatic Trace ID injection
+## Automatic log injection
 
 Use one of the following options to inject Ruby trace information into your logs:
 
 - [Automatic Trace ID Injection for Rails Applications using Lograge is recommended](?tab=lograge).
 - [Automatic Trace ID Injection for default Rails Applications](?tab=default).
 
+If you haven't done so already, we recommend configuring the Java tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the smoothest
```

Comment:
```suggestion
If you haven't done so already, we recommend configuring the Ruby tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the smoothest
```